src/core/server/saved_objects/service/lib/repository.test.js

```diff
@@ -417,6 +417,27 @@ describe('SavedObjectsRepository', () => {
       });
     });
 
+    it('accepts custom refresh settings', async () => {
```

Comment:
can you add tests for the default behaviour for `refresh` for all the methods?

Reply:
Sure.
They're tested by other existing tests, in a non explicit manner, but happy to add  explicit ones. :)